### DISCUSSION TITLE
Would anyone be interested in drag and drop?

### DISCUSSION THREAD

**Discussion by rappasoft** [rappasoft/laravel-livewire-tables#349] [Show and tell]
I have created a (_very alpha_) drag and drop addition that I'm not sure is worth keeping.

When enabled, it disables all other features because all the items need to be on one page without messing with the filters or sorts. These are just limitations of the livewire sortable plugin.

It actually works really well if you just have a list of information you want to be able to reorder or do bulk action on.

I'm not sure all the added configuration I had to add is going to be worth the issues and requests that will arise.

But anyway, here is a demo:

https://user-images.githubusercontent.com/6026504/122629392-980d3a00-d08a-11eb-8590-cba8b3fb77d0.mov

This also has a demo of the upcoming `column selector` for v1.10

**Comment by rappasoft** [comment]
I've made it even better without losing any other functionality:


https://user-images.githubusercontent.com/6026504/122648811-d0e3f800-d0f8-11eb-8c8e-3b47ceb43798.mov

**Reply by tuluong-vl** [reply]
Bravo @rappasoft 
I think it should have:

1. a $sortColumnName to map with table's column
2. only enabled ordering with $paginationEnabled = false
3.  without any sortable columns except the $sortColumnName

**Comment by Todomed** [comment]
if I am interested in the functionality of drag and drop the rows. Now I have a drawback: when there are many records it gets slow. A good option is that this function has pagination so that it can work without problem.

**Reply by rappasoft** [reply]
You can't because the Javascript drag and drop packages require all the elements to be on one page so it knows the order of everything.

**Comment by dgillier** [comment]
Hello,
I'm trying to use the reordering feature of this package, and expecting drag and drop (as there are handles in the datatable when I click on reorder).
But it looks like there is no drag and drop ???
I'm I missing something ?

Thanks, Denis

**Comment by dgillier** [comment]
RTFM (LOL )
"You must have [Livewire Sortable](https://github.com/livewire/sortable) or [Livewire Sortable.js](https://github.com/nextapps-be/livewire-sortablejs) installed for this feature."
